module_1/package.json

```diff
@@ -6,12 +6,15 @@
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "@fortawesome/fontawesome-svg-core": "^6.4.2",
     "@fortawesome/free-solid-svg-icons": "^6.4.2",
+    "@fortawesome/react-fontawesome": "^0.2.0",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.5.1",
+    "focus-trap-react": "^10.2.3",
```

Comment:
Looks like you forgot to add FocusTrap component to your Dialog component. I think you can easily add it or remove the package